arm-graphrbac/1.6/swagger/graphrbac.json

```diff
@@ -1254,6 +1546,54 @@
       },
       "description": "Role assignment list operation result."
     },
+    "KeyCredentialListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/KeyCredential"
+          },
+          "description": "KeyCredential list."
+        }
+      },
+      "description": "KeyCredential list operation result."
+    },	
+	"KeyCredentialsUpdateParameters": {
```

Comment:
This is same as the KeyCredentialUpdateParameters and we decided to keep it like that as per our offline conversation.